src/algebra/order/absolute_value.lean

```diff
@@ -152,14 +142,17 @@ section linear_ordered_ring
 variables {R S : Type*} [semiring R] [linear_ordered_ring S] (abv : absolute_value R S)
 
 /-- `absolute_value.abs` is `abs` as a bundled `absolute_value`. -/
-@[simps]
 protected def abs : absolute_value S S :=
 { to_fun := abs,
   nonneg' := abs_nonneg,
   eq_zero' := λ _, abs_eq_zero,
   add_le' := abs_add,
   map_mul' := abs_mul }
 
+-- I'm not sure how to get `simps` working with this.#check
```

Comment:
IIRC you need to write something like:
```lean
def absolute_value.simps.apply (f : absolute_value R S) : R → S := coe_fn f

initialize_simps_projections absolute_value (to_fun → apply)
```
See also [note custom simps projection](https://leanprover-community.github.io/mathlib_docs/notes.html#custom%20simps%20projection)